packages/react/src/PricingOptions/PricingOptions.tsx

```diff
@@ -382,35 +396,50 @@ type ResponsiveExpandableProps = {
   wide: boolean
 }
 
-const defaultExpanded: ExpandedProp = {
-  narrow: false,
-  regular: true,
-  wide: true,
-}
-
 type ValidFeatureListChildren = {
   Heading: React.ReactElement<PricingOptionsFeatureListGroupHeadingProps> | null
   Items: React.ReactElement<PricingOptionsFeatureListItemProps>[]
 }[]
 
 const PricingOptionsFeatureList = forwardRef<HTMLDivElement, PricingOptionsFeatureListProps>(
-  (
-    {
-      children,
-      className,
-      'data-testid': testId,
-      hasDivider = true,
-      expanded = defaultExpanded,
-      accordionAs = 'h4',
-      ...rest
-    },
-    ref,
-  ) => {
-    const runOnce = React.useRef(false)
-    const [isAccordionOpen, setIsAccordionOpen] = React.useState<boolean>(false)
-    const {allFeatureListsExpanded, updateFeatureListExpanded} = usePricingOptions()
+  ({children, className, 'data-testid': testId, hasDivider = true, expanded, accordionAs = 'h4', ...rest}, ref) => {
+    const {
+      allFeatureListsExpanded,
+      updateFeatureListExpanded,
+      featureListUserInteracted,
+      setFeatureListUserInteracted,
+    } = usePricingOptions()
+    const {isMedium: isRegular, isXLarge: isWide, isSmall: isNarrow} = useWindowSize()
+
+    React.useEffect(() => {
+      if (expanded === undefined || typeof expanded === 'object') {
+        setFeatureListUserInteracted(false)
+      }
+    }, [isRegular, isWide, isNarrow, setFeatureListUserInteracted, expanded])
+
+    const shouldBeOpen = React.useMemo(() => {
+      if (featureListUserInteracted) {
+        return allFeatureListsExpanded
+      }
+
+      // Important: Fixes layout shift by ensuring that explicit values are preserved.
+      if (typeof expanded === 'boolean') {
+        return expanded
+      }
+      // If undefined, should follow some default responsive logic
+      if (expanded === undefined) {
+        if (isWide) return true
+        if (isRegular) return true
+        return false
+      }
 
-    const {isMedium: isRegular, isXLarge: isWide} = useWindowSize()
+      // finally handle responsive objects, and go from largest to smallest viewports
+      const {narrow, regular, wide} = expanded
+      if (isWide) return wide
+      if (isRegular) return regular
+      if (isNarrow) return narrow
```

Comment:
Minor thing, but this check is redundant.

If `isNarrow` is truthy then we return `narrow`. If it's not, we still return `narrow`.
```suggestion
```